sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1195,6 +1246,7 @@ private static class BigtableWriterFn
       this.writeOptions = writeOptions;
       this.failures = new ConcurrentLinkedQueue<>();
       this.id = factory.newId();
+      LOG.info("Created Bigtable Write Fn with writeOptions {} ", writeOptions);
```

Comment:
I'm not sure if this should be info. WriteOptions is an internal class, so logging it by default is not useful to endusers. I would drop this to debug